database1/seed.js

```diff
@@ -0,0 +1,32 @@
+const faker = require('faker');
+
+const furnitureArr = [];
+
+//current S3 library holds 101 photos. Will adapt variable when I have more photos.
+const photoLibrarySize = 101;
+//At 1 million there is about 1 second delay to execute function. Plan to seed db in increments of 1m to start --> run createProductEntry to generate 1m, seed db, run createProductEntry again to reach 10m
```

Comment:
Interesting way to balance memory usage! This should work, but I also have an interesting thought about it. You're still creating an array with a million things in it, so what if you have to perform this in an instance that has very little RAM? I think it hits a cap of how scaled down it can get. Also, what if you don't want 10,000,000 results, but 100,000,000 or 1,000,000,000? You'd have to be generating 1,000,000 results a lot of times, which means you'd be manually running your seed a lot of times, right?

Reply:
Thinking about it! Thank you!